### PR TITLE
Update get-started.md for Ember

### PR DESCRIPTION
**Pull request by amyrlam**
Correct command, should be `ember serve` re: https://guides.emberjs.com/release/getting-started/quick-start/#toc_create-a-new-application

IS:

```
$ ember start

The specified command start is invalid. For available options, see `ember help`.
```

### CODE REVIEW

**Comment by jonniebigodes** [comment]
That's great and thank you very much for addressing the feedback. Looking forward to the updates.

File: content/intro-to-storybook/ember/en/get-started.md [hunk 1]
```diff
@@ -42,11 +42,10 @@ Now we can quickly check that the various environments of our application are wo
 ember test --server
 
 # Run the frontend app proper on port 4200:
-ember start
+yarn run start
 
 # Start the component explorer on port 6006:
-npm run storybook-dev
-
+yarn run storybook-dev
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/intro-to-storybook/ember/en/get-started.md:48]: @amyrlam to give you a bit of context on the `yarn storybook-dev` command when you pull in the template, you have the barebones structure to get started with the tutorial (CSS, Storybook with default examples, etc...). For the time being, we're leaving in the Ember app empty and focusing on the small pieces, building momentum until the Screen chapter, that's when you or any other reader will actually work on the app itself by modifying the `app/templates/aplication.hbs` file to get a working application. Also in order to get a functional Storybook working with Ember you'll first need to build the application before starting Storybook, hence why that script. We're aware that this is a bit cumbersome in terms of user experience, but something that we've been discussing with the Storybook Ember maintainers in order to get fixed and improve the overall workflow similar to other frameworks, but that's still up for debate and if you are willing to work with us and help us please, by all means, check the Storybook monorepo and give us a hand. We'd love any help you could provide us in this matter!

**Comment by jonniebigodes** [comment]
Going to merge this! @amyrlam thanks for helping us out and catching those typos, we really appreciate it!

Feel free to reach out to us with further issues you might encounter while reading the tutorials! We welcome those.

Have a great week!

Stay safe

File: content/intro-to-storybook/ember/en/get-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amyrlam) [reply content/intro-to-storybook/ember/en/get-started.md:48]: @jonniebigodes Thanks for the explanation here and the merge, I've been out with RSI, hopefully the GitHub "out of office message" worked! 

It's definitely on my radar to check out the code behind Storybook for Ember more, hopefully 🔜

**Pull request merged**